drivers/hmis/app/graphql/mutations/submit_assessment.rb

```diff
@@ -53,6 +53,9 @@ def resolve(input:, assessment_lock_version: nil)
 
       has_already_been_submitted = assessment.persisted? && !assessment.in_progress?
 
+      assessment.updated_by = current_user
+      assessment.created_by = current_user unless has_already_been_submitted
```

Comment:
In the script to backfill the users from the versions we are using the "true" user if it exists. I believe that means that we use the original user if there was an impersonation. But we don't use the impersonating user here.

Does that inconsistency matter?

Reply:
Hm thanks for pointing that out, yes, I should fix that. My understanding is that we want the new `created_by` and `updated_by` attributes to match whatever the current behavior is on `load_last_user_from_versions` and `load_created_by_user_from_versions`, which appears to load the true user. https://github.com/greenriver/hmis-warehouse/blob/release-142/drivers/hmis/app/graphql/sources/paper_trail_versions.rb#L26-L39